.travis.yml

```diff
@@ -2,9 +2,17 @@ language: node_js
 node_js:
   - "node"
 
-after_success:
-  - git config --global user.name "marwin1991"
-  - git config --global user.email "peter.zmilczak@gmail.com"
-  - git remote rm origin
-  - git remote add origin https://marwin1991:${GH_TOKEN}@github.com/libamtrack/web.git
-  - npm run deploy
+install:
+  - npm install
+script:
+  - npm run build
+
+deploy:
+  provider: pages
+  skip-cleanup: true
+  github-token: $GH_TOKEN
```

Comment:
Where is GH_TOKEN being defined ?

Reply:
in travis/settings/Environment Variables